### DISCUSSION TITLE
How to use [fix](https://github.com/aio-libs/aiopg/issues/678#issuecomment-667908402) with pytest.

### DISCUSSION THREAD

**Discussion by heckad** [pytest-dev/pytest#9090] [Q&A]
I tried to add this to conftest and this didn't solve problem. Pytest still uses `ProactorEventLoop`.

**Comment by RonnyPfannschmidt** [comment]
this is a issue with pytest-asyncio - as far as i understood you need to set up its event loop replacement policy (it uses separate event loops)

see its documentation

**Reply by heckad** [reply]
We don't use `pytest-asyncio` only pytest and aiopg under windows. Ower tests fails with `NotImplementedError` because pytest used ProactorEventLoop instead `SelectorEventLoop`. How to change event loop?

**Reply by RonnyPfannschmidt** [reply]
pytest itself does not configure the event loop, i suspect that by the time your conftest is imported, a event loop was acquired somewhere
if you use asyncio, i strongly recommend using a pytest-plugin that configures it for testing

**Reply by heckad** [reply]
I just need to use loop fixture to set the desired event loop. Problem solved.
